### ISSUE TITLE
The newest RoboWareStudio could not format the code

### ISSUE THREAD

**Issue by lily0101** [tonyrobotics/RoboWare#32]
I download the newest RoboWareStudio, roboware-studio_0.7.2-1503540841_amd64.deb, and I setup it. But the formatting function is not available. when I click "ctrl + K + F", the code is still unformatted. I check the keybindings, it does not have any problem.

**Comment by tulayang** [comment]
Indeed, the formatting function is still not available. This feature will be added in the future release.

**Comment by kxhit** [comment]
Looking forward to formatting function! Please let us know if the feature is added in a future release. Thanks!
